test/controllers/comment_controller_test.exs

```diff
@@ -87,6 +87,22 @@ defmodule CodeCorps.CommentControllerTest do
 
       assert json_response(conn, 401)
     end
+
+    @tag :authenticated
+    test "does not create resource and renders 401 when not authorized", %{conn: conn} do
```

Comment:
That's testing authentication but not authorization, I believe. The package we're using incorrectly returns 401 for unauthorized rather than 403.